pkg/action/status.go

```diff
@@ -27,6 +27,11 @@ type Status struct {
 	cfg *Configuration
 
 	Version int
+
+	// If true, display description to output format,
+	// only affect print type table.
+	// This flag will be true by default in Helm 4.
```

Comment:
This flag will be true by default in Helm 4. --> This flag will removed in Helm 4 and description will be output by default.